tests/acl/null_route/test_null_route_helper.py

```diff
@@ -68,35 +68,59 @@
+    Remove DATAACL and EVERFLOW to free TCAM resources.
+    The change is written to configdb as we don't want DATAACL recovered after reboot
     """
-    TABLE_NAME = "DATAACL"
-    lines = rand_selected_dut.shell(cmd="show acl table {}".format(TABLE_NAME))['stdout_lines']
-    data_acl_existing = False
-    for line in lines:
-        if TABLE_NAME in line:
-            data_acl_existing = True
-            break
-    if not data_acl_existing:
+    TABLE_NAME_1 = "DATAACL"
+    TABLE_NAME_2 = "EVERFLOW"
+    for duthost in duthosts:
+        lines = duthost.shell(cmd="show acl table {}".format(TABLE_NAME_1))['stdout_lines']
+        data_acl_existing = False
+        for line in lines:
+            if TABLE_NAME_1 in line:
+                data_acl_existing = True
+                break
+
+        lines = duthost.shell(cmd="show acl table {}".format(TABLE_NAME_2))['stdout_lines']
+        everflow_acl_existing = False
+        for line in lines:
+            if TABLE_NAME_2 in line:
+                everflow_acl_existing = True
+                break
+
+        if data_acl_existing:
+            # Remove DATAACL
+            logger.info("Removing ACL table {}".format(TABLE_NAME_1))
+            rand_selected_dut.shell(cmd="config acl remove table {}".format(TABLE_NAME_1))
+        if everflow_acl_existing:
+            # Remove EVERFLOW
+            logger.info("Removing ACL table {}".format(TABLE_NAME_2))
+            rand_selected_dut.shell(cmd="config acl remove table {}".format(TABLE_NAME_2))
+
+    if not data_acl_existing and not everflow_acl_existing:
         yield
         return
-    # Remove DATAACL
-    logger.info("Removing ACL table {}".format(TABLE_NAME))
-    rand_selected_dut.shell(cmd="config acl remove table {}".format(TABLE_NAME))
+
     yield
     # Recover DATAACL
     config_db_json = "/etc/sonic/config_db.json"
     output = rand_selected_dut.shell("sonic-cfggen -j {} --var-json \"ACL_TABLE\"".format(config_db_json))['stdout']
-    try:
-        entry = json.loads(output)[TABLE_NAME]
+    entry_json = json.loads(output)
+    if TABLE_NAME_1 in entry_json:
+        entry = entry_json[TABLE_NAME_1]
+        cmd_create_table = "config acl add table {} {} -p {} -s {}"\
+            .format(TABLE_NAME_1, entry['type'], ",".join(entry['ports']), entry['stage'])
+        logger.info("Restoring ACL table {}".format(TABLE_NAME_1))
+        rand_selected_dut.shell(cmd_create_table)
+
+    if TABLE_NAME_2 in entry_json:
+        entry = json.loads(output)[TABLE_NAME_2]
         cmd_create_table = "config acl add table {} {} -p {} -s {}"\
-            .format(TABLE_NAME, entry['type'], ",".join(entry['ports']), entry['stage'])
-        logger.info("Restoring ACL table {}".format(TABLE_NAME))
+            .format(TABLE_NAME_2, entry['type'], ",".join(entry['ports']), entry['stage'])
+        logger.info("Restoring ACL table {}".format(TABLE_NAME_2))
         rand_selected_dut.shell(cmd_create_table)
```

Comment:
The logic looks good to me. It should be better if you put `TABLE_NAME_1` and `TABLE_NAME_2` in a list and walk through them in a loop.

Reply:
Addressed the comments. Please check.